WalletWasabi/Microservices/ProcessInvoker.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.Microservices
+{
+	public class ProcessInvoker
+	{
+		public async Task<(string response, int exitCode)> SendCommandAsync(ProcessStartInfo startInfo, CancellationToken token, Action<StreamWriter>? standardInputWriter = null)
+		{
+			using var processAsync = new ProcessAsync(startInfo);
+
+			if (standardInputWriter is { })
+			{
+				processAsync.StartInfo.RedirectStandardInput = true;
+			}
+
+			processAsync.Start();
+
+			if (standardInputWriter is { })
+			{
+				standardInputWriter(processAsync.StandardInput);
```

Comment:
I regret having done this hack. If you continue improving the ProcessBridge family please remove this and promote your `ProcessAsync` to a higher level of abstraction by adding something like `processAsync.Send(string input)` or something like that.

Reply:
ok